AdobeStockImageAdminUi/Test/Mftf/Test/UserSearchesStockImagesByKeywords.xml

```diff
@@ -0,0 +1,94 @@
+        <click selector="{{AdobeStockSection.searchButton}}" stepKey="applySearchWithTopSecret"/>
+        <waitForPageLoad stepKey="waitForSearchWithTopSecret"/>
+        <dontSeeElement selector="//div[@class='masonry-image-column' and contains(@data-repeat-index, '0')]//img[@src='{$getUrlFromFirstImageWithoutSearch}']" stepKey="checkFirstImageNotFoundAfterSearchTopSecret"/>
+
+        <grabAttributeFrom selector="div[class='masonry-image-column'][data-repeat-index='0'] img" userInput="src" stepKey="getUrlFromFirstImageAfterSearchTopSecret"/>
+        <fillField selector="{{AdobeStockSection.searchInput}}" userInput="%0F" stepKey="searchForThirdSymbols"/>
+        <click selector="{{AdobeStockSection.searchButton}}" stepKey="applySearchWithThirdSymbols"/>
+        <waitForPageLoad stepKey="waitForSearchWithThirdSymbols"/>
+        <dontSeeElement selector="//div[@class='masonry-image-column' and contains(@data-repeat-index, '0')]//img[@src='{$getUrlFromFirstImageAfterSearchTopSecret}']" stepKey="checkFirstPopularImageNotFoundAfterSearchTopSecret"/>
+        <dontSeeElement selector="//aside[contains(@class, 'modal-popup')]//div[@class='modal-content']//div[text()='Something went wrong.']" stepKey="errorModalIsAbsentAfterSearchThirdSymbols"/>
+        <dontSeeElement selector="//div[contains(@id, 'adobe-stock-images-search-modal')]//div[@class='spinner']" stepKey="spinnerIsAbsentAfterSearchThirdSymbols"/>
+
+        <fillField selector="{{AdobeStockSection.searchInput}}" userInput="chest band heart rate" stepKey="searchForChestBand"/>
+        <click selector="{{AdobeStockSection.searchButton}}" stepKey="applySearchWithChestBand"/>
+        <waitForPageLoad stepKey="waitForSearchWithChestBand"/>
+        <dontSeeElement selector="//div[@class='masonry-image-column' and contains(@data-repeat-index, '0')]//img[@src='{$getUrlFromFirstImageWithoutSearch}']" stepKey="checkFirstImageNotFoundAfterSearchChestBand"/>
+
+        <grabAttributeFrom selector="div[class='masonry-image-column'][data-repeat-index='0'] img" userInput="src" stepKey="getUrlFromFirstImageAfterSearchChestBand"/>
+        <fillField selector="{{AdobeStockSection.searchInput}}" userInput="mi band heart rate" stepKey="searchForMiBand"/>
+        <click selector="{{AdobeStockSection.searchButton}}" stepKey="applySearchWithMiBand"/>
+        <waitForPageLoad stepKey="waitForSearchWithMiBand"/>
+        <dontSeeElement selector="//div[@class='masonry-image-column' and contains(@data-repeat-index, '0')]//img[@src='{$getUrlFromFirstImageAfterSearchChestBand}']" stepKey="checkFirstImageNotFoundAfterSearchMiBand"/>
+        <dontSeeElement selector="//aside[contains(@class, 'modal-popup')]//div[@class='modal-content']//div[text()='Something went wrong.']" stepKey="errorModalIsAbsentAfterSearchChestBand"/>
+        <dontSeeElement selector="//div[contains(@id, 'adobe-stock-images-search-modal')]//div[@class='spinner']" stepKey="spinnerIsAbsentAfterSearchChestBand"/>
```

Comment:
Test in general looks two complex and contains too many steps. I would suggest create a few meaningful action groups and use them instead.